index/index.js

```diff
@@ -0,0 +1,8 @@
+
+const ipcRenderer = require('electron').ipcRenderer;
+
+var camera, scene, renderer,rightArm;
+ipcRenderer.on('open-image', (event,image) => {
+   console.log(image)
```

Comment:
This may just be innate to electron launching, but why is there a console.log() here and what does it print out?

Reply:
This is a proof of concept for image use. Prints out the image path in chrome developer tools. Will refactor later.